test/runtests.jl

```diff
@@ -21,6 +21,10 @@ end
     @info "Building missingdocs/make.jl"
     include("missingdocs/make.jl")
 
+    # Test invalid links (linkcheck option)
+    @info "Building invalidlinks/make.jl"
+    include("invalidlinks/make.jl")
```

Comment:
After writing up https://github.com/JuliaDocs/Documenter.jl/issues/2329#issuecomment-1794116134, I now  think that we should actually move these tests into `online_linkcheck.jl`, since while they fail, they do try to talk to the internet, which can be flaky in other ways too probably.

Reply:
Makes sense!

I'll clean up the other stuff – let me know about https://github.com/JuliaDocs/Documenter.jl/pull/2330#discussion_r1384076737; and then see if I can move it over.

Reply:
OK, I moved it over to `test/online_linkcheck.jl` (and renamed `test/invalidlinks/make.jl` -> `test/docstring_links/make.jl`)